src/lib/CollectStore.js

```diff
@@ -145,7 +145,7 @@ export default class CollectStore {
               arguments: '0 0 0 * * *',
               worker: 'konnector',
               worker_arguments: {
-                konnector: connection.konnector.attributes.slug,
+                konnector: connection.konnector.slug,
```

Comment:
If I throw it, it will appear in the "short lived" notification and I prefer to have it in the console. I will do both.